src/utils/validation.ts

```diff
@@ -7,11 +7,14 @@ import { AUTH } from '../langs/constants';
 const {
   USERNAME_TOO_LONG_ERROR,
   USERNAME_TOO_SHORT_ERROR,
+  USERNAME_SPECIAL_CHARACTERS_ERROR,
   INVALID_EMAIL_ERROR,
   PASSWORD_EMPTY_ERROR,
   EMPTY_EMAIL_ERROR,
 } = AUTH;
 
+const USER_NAME_REGEX = /["<>^%\\\s]+/;
```

Comment:
Should this go in `@graasp/sdk` since you reuse it in account?
Why is it different from the backend's regex?

Reply:
Yeah, it makes sense to put in the SDK. I'll do that.